File: Server/Cinema/CinemaApp.Domain/Entities/MovieSession.cs
namespace CinemaApp.Domain.Entities
{
    public class MovieSession
    {
        public long Id { get; set; }
        public DateTime StartShowingTime { get; set; }
        public DateTime EndShowingTime { get; set; }
        public int? MovieId { get; set; }
        public Movie Movie { get; set; }
        public int? HallId { get; set; }
        public Hall Hall { get; set; }
        public ICollection<TicketPrice> TicketPrices { get; set; }
        public ICollection<Ticket> Tickets { get; set; }
    }
}

File: Client/cinema-app/src/app/core/models/movie-session/movie-session.ts
import { TicketPrice } from '@core/models/ticket-price/ticket-price';

export interface MovieSession {
  id: number,
  movieId: number,
  hallId: number,
  startShowingTime: Date,
  endShowingTime: Date,
  ticketPrices: TicketPrice[]
}
